KsApi/models/templates/ProjectTemplates.swift

```diff
@@ -21,6 +21,7 @@ extension Project {
       stateChangedAt: NSDate().timeIntervalSince1970 - 60.0 * 60.0 * 24.0 * 15.0
     ),
     id: 1,
+    liveStreams: [.template],
```

Comment:
i'd suggest using an empty array so that in tests we specifically populate a live stream for those cases.